app/src/main/java/io/pumpkinz/pumpkinreader/NewsDetailFragment.java

```diff
@@ -77,6 +77,10 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
         newsDetail = (RecyclerView) view.findViewById(R.id.news_detail);
+        newsDetail.getItemAnimator().setRemoveDuration(100);
+        newsDetail.getItemAnimator().setChangeDuration(0);
```

Comment:
`setChangeDuration` is when we touch the comment item.
I set it to 0 to make it snappier.
The default value will make the comment item blinks.